### DISCUSSION TITLE
Remove gap between expandable sections?

### DISCUSSION THREAD

**Discussion by designbyadrian** [GitbookIO/community#497] [2. Published content] [answered]
Hello, in [this example](https://docs.gitbook.com/content-editor/blocks/expandable), the expandable sections are grouped nicely into one block.

I've tried adding a new expandable section using the + button, and also tried removing newlines, as well as gaps in the markdown (github sync) to no avail. I only get a full new line between each expandable section.

How is this achieved on the demo page?

**Comment by SamyPesse** [comment, marked as answer]
This is something that is only done on the new version of our published content renderer at the moment (used for our documentation website, and coming soon to all users).

We are looking at applying the same styling in the application.

In short: you don't need to do anything, create expandables, they will be grouped automatically when publishing the content, and in the future it'll also work in the app :)

**Reply by designbyadrian** [reply]
OIC! The _published_ site will look slightly different! I'll keep my eye on it for when it's time.

Thank you for the prompt reply!

**Reply by j0sh** [reply]
This is super confusing UX if the editor (and preview) does not match the actual output. I lost quite a bit of of time with this because I thought that would be the final rendering and kept trying to figure out how to collapse the gap.

**Reply by nicolas-kuyumciyan-tracebit** [reply]
@SamyPesse This discrepancy is still a problem almost a year later.  Are there plans on updating the rendering on the app to match published content?
